### PR TITLE
docs: refactor contributing guide and add how to run a single test guide

### PR DESCRIPTION
**Pull request by tisonkun**
This closes https://github.com/apache/incubator-kvrocks/issues/1450.

For reviewing, I suggest you just open the [preview page](https://github.com/tisonkun/incubator-kvrocks-website/blob/doc-go-test/community/contributing.md) and review the final result.

### CODE REVIEW

File: community/contributing.md [hunk 1]
```diff
@@ -25,47 +26,57 @@ To help you familiarise yourself with the source tree, we have listed some impor
...
 
-As you find tasks to start participating in the community, you will gradually learn about the workings and practices of the community.
-Here are some tips to help you feel more comfortable in the community:
+As you find tasks to start participating in the community, you will gradually learn about the workings and practices of the community. Here are some tips to help you feel more comfortable in the community:
+
 - Try to express your intentions in as many words as possible, and do not be afraid of words.
 - Talk to other members to get support before starting a new idea.
 
-After being involved in the community for a while, some members may want to learn how to become a committer, who has write access to the project (this is limited to merging PRs that have been approved and passed the CI, as kvrocks protects the main branch), and the ability to review PRs from others.
+After being involved in the community for a while, some members may want to learn how to become a committer, who has write access to the project (this is limited to merging PRs that have been approved and passed the CI, as kvrocks protects the unstable branch), and the ability to review PRs from others.
```
Comment (torwig) [inline community/contributing.md:78]: I think in this line the  `kvrocks` word also should be capitalized.

Comment (tisonkun) [reply community/contributing.md:78]:
```suggestion
After being involved in the community for a while, some members may want to learn how to become a committer, who has write access to the project (this is limited to merging PRs that have been approved and passed the CI), and the ability to review PRs from others.
```

Comment (tisonkun) [reply community/contributing.md:78]: We can even simplify the sentence :D

**Review by torwig — APPROVED** [review]
LGTM

**Review by git-hulk — APPROVED** [review]

**Comment by enjoy-binbin** [comment]
please find a way (or place) that also mention `deleteOnExit` or other flags?

**Pull request merged**